contracts/Strategy.sol

```diff
@@ -784,10 +768,33 @@ contract Strategy is BaseStrategy, DydxFlashloanBase, ICallee {
         Account.Info memory account,
         bytes memory data
     ) public override {
-        (bool deficit, uint256 amount, uint256 repayAmount) = abi.decode(data, (bool, uint256, uint256));
+        (bool deficit, uint256 amount) = abi.decode(data, (bool, uint256));
         require(msg.sender == SOLO);
+        require(sender == address(this));
+        uint256 repayAmount = amount.add(2);
+
+        uint256 bal = want.balanceOf(address(this));
+        require(bal >= amount); // to stop malicious calls
+
+        IWETH(weth).withdraw(bal);
+
+        //if in deficit we repay amount and then withdraw
+        if (deficit) {
+
+            cToken.repayBorrow{value: amount}();
+
+            //if we are withdrawing we take more to cover fee
```

Comment:
question:

which fee are we trying to cover here? is that the reason for `.add(2);` in repayAmount?

Reply:
the dydx extra fee